python/sedona/maps/SedonaKepler.py

```diff
@@ -0,0 +1,58 @@
+from keplergl import KeplerGl
+import geopandas as gpd
```

Comment:
You introduced 4 new files in this PR. All new files must have Apache License header. Please refer to existing python files in Sedona.

Reply:
Added